tools/unity-avatar-exporter/Assets/Editor/AvatarExporter/AvatarExporter.cs

```diff
@@ -638,32 +640,32 @@ class MaterialData {
         // close the preview avatar scene and go back to user's previous scene when export project windows close
         ClosePreviewScene();
     }
-    
-    // The High Fidelity FBX Serializer omits the colon based prefixes. This will make the jointnames compatible.
+
+    // The Overte FBX Serializer omits the colon based prefixes. This will make the jointnames compatible.
     static string removeTypeFromJointname(string jointName) {
         return jointName.Substring(jointName.IndexOf(':') + 1);
     }
-    
-    static bool WriteFST(string exportFstPath, string projectName, float scale) {        
+
+    static bool WriteFST(string exportFstPath, string projectName, float scale) {
         // write out core fields to top of fst file
         try {
-            File.WriteAllText(exportFstPath, "exporterVersion = " + AVATAR_EXPORTER_VERSION + "\nname = " + projectName + 
-                                             "\ntype = body+head\nscale = " + scale + "\nfilename = " + assetName + 
+            File.WriteAllText(exportFstPath, "exporterVersion = " + AVATAR_EXPORTER_VERSION + "\nname = " + projectName +
+                                             "\ntype = body+head\nscale = " + scale + "\nfilename = " + assetName +
                                              ".fbx\n" + "texdir = textures\n");
-        } catch { 
-            EditorUtility.DisplayDialog("Error", "Failed to write file " + exportFstPath + 
+        } catch {
+            EditorUtility.DisplayDialog("Error", "Failed to write file " + exportFstPath +
                                         ". Please check the location and try again.", "Ok");
             return false;
         }
-        
+
         // write out joint mappings to fst file
         foreach (var userBoneInfo in userBoneInfos) {
             if (userBoneInfo.Value.HasHumanMapping()) {
-                string hifiJointName = HUMANOID_TO_HIFI_JOINT_NAME[userBoneInfo.Value.humanName];
-                File.AppendAllText(exportFstPath, "jointMap = " + hifiJointName + " = " + removeTypeFromJointname(userBoneInfo.Key) + "\n");
+                string vircJointName = HUMANOID_TO_OVERTE_JOINT_NAME[userBoneInfo.Value.humanName];
```

Comment:
Why `vircJointName`? Aren't you creating a new variable?
```suggestion
                string overteJointName = HUMANOID_TO_OVERTE_JOINT_NAME[userBoneInfo.Value.humanName];
```

Reply:
Ah missed that one. The hifiJointName -> vircJointName was from where I contributed to Virc fork before joining overte. Overte hadn't synced those changes so it was just carry over from there